Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -328,9 +327,7 @@ def fetch_compiler(value, source)
   end
 
   def check_for_compiler_universal_support
-    if homebrew_cc =~ GNU_GCC_REGEXP
-      raise "Non-Apple GCC can't build universal binaries"
-    end
+    raise "Non-Apple GCC can't build universal binaries" if homebrew_cc =~ GNU_GCC_REGEXP
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.